gateway-api/users-api/api/get-user.spec.js

```diff
@@ -0,0 +1,17 @@
+const app = require('../app.js');
+const should = require('should');
+const supertest = require('supertest');
+
+describe('/get-user', () => {
+    it('should return a users individual data', done => {
+      supertest(app)
+      .get('/get-user')
+      .end((err, res) => {
+        res.text.should.equal(
```

Comment:
I've tried this and it gives me a TypeError stating that it cannot read property 'should' of undefined.